README.md

```diff
@@ -42,6 +42,26 @@ Include `mh-opsworks-recipes` in your node's `run_list`:
 
 * Dan Collis-Puro - [djcp](https://github.com/djcp)
 
+## Development
+
+* Create a branch off the develop branch
+* Do your work and test it in a running mh-opsworks cluster
+* Be sure to update the "To be released" section of the CHANGELOG.md file with
+  a description of your change, and whether or not it requires a manual recipe
+  invocation. If your change runs as part of the deploy lifecycle, it will be run
+  the next time the main app is deployed to a cluster and therefore does not
+  require a manual chef recipe invocation.
+* Rebase against develop and submit a pull request off of develop.
+
+## Releasing
+
+* Update CHANGELOG.md to collapse all "To be released" changes under a new
+  version, following [semver](http://semver.org).
+* Rebase develop onto master.
+* Merge develop into master.
+* Tag the release on master.
+* Push the tag to the remote via `git push --tags`
+
```

Comment:
Only if there are updates to be released for chef provisioning. Also, this might be a thing where we merge and tag the chef recipes the night before a release - no harm in that because all important matterhorn clusters will be using explicit tags.